src/components/studyBrowser/DragPreview.js

```diff
@@ -1,10 +1,29 @@
-import React, { Component } from 'react';
+import React, { PureComponent, memo } from 'react';
 import { DragLayer } from 'react-dnd';
 import PropTypes from 'prop-types';
 import { ThumbnailEntry } from './ThumbnailEntry';
 import './DragPreview.styl';
 
-function collect(monitor) {
+let subscribedToOffsetChange = false;
+let dragPreviewRef = null;
+
+const collector = monitor => {
```

Comment:
This is cool, but is there a good reason we're not using the web's native/built-in `setDragImage`?

https://developer.mozilla.org/en-US/docs/Web/API/DataTransfer/setDragImage

Reply:
Or maybe that's what's being used under the hood here? I'm unfamiliar with the DnD library we're using. If this is the path it sets out for us, then I'm okay with a merge. Just want to make sure we're not adding unnecessary complexity.

Reply:
https://github.com/react-dnd/react-dnd/blob/a73227c4512b083068a13f43e8527bbcdfa53240/packages/core/html5-backend/src/HTML5Backend.ts#L387

Looks like it will use it if it's set?